seed/tests/api/data/covered-buildings-sample.csv

```diff
@@ -1,513 +0,0 @@
-UBI,GBA,BLDGS,Address,Owner,City,State,Zip,Property Type,AYB_YearBuilt
```

Comment:
Yes. We already have the file in seed/tests/data, and that's where all the tests were pointing to.